app/views/admin/loans/questionnaires/breakeven_rampup/_rampup_table.html.slim

```diff
@@ -0,0 +1,26 @@
+tr.group-content
+  td
+    td.text-center colspan=periods
+      = units
```

Comment:
Not sure if this is a great way to do this or not? I could have done this instead with two rows, but less padding?